flink-table/flink-sql-parser-hive/src/test/java/org/apache/flink/sql/parser/hive/FlinkHiveSqlParserImplTest.java

```diff
@@ -35,15 +35,12 @@ protected SqlParserImplFactory parserImplFactory() {
 	}
 
 	// overrides test methods that we don't support
-	@Override
```

Comment:
What about changing to use `@Ignore` +	`@Test` annotations and also update the comment above?
This can avoid the IDEA warns this method is never used.